governance.md

```diff
@@ -0,0 +1,201 @@
+# Governance
+
+This document describes the rules and governance of the project. It is a slightly modified version of the [Prometheus Governance](https://prometheus.io/governance/#governance-changes).
+
+It is meant to be followed by all the developers of the Prometheus Operator project and the Prometheus Operator community. Common terminology used in this governance document are listed below:
+
+* **Maintainers Team**: A core Prometheus Operator team that have owner access to http://github.com/prometheus-operator organization and all projects within it. Current list is available [here][maintainers-doc].
+
+* **Triage Team**: Contributors who does not belong to Maintainer's team, but has `Triage` GitHub role on [Prometheus Operator](https://github.com/prometheus-operator) repository allowing to change GitHub issues and PRs statuses and labels.
+They are listed [here](https://thanos.io/tip/thanos/maintainers.md/#triage).
```

Comment:
can we link to our doc?

Reply:
Urgh, looks like I missed a bunch of `thanos` (note lowercase, I seem to have searched for `Thanos`) mentions. Fixing.

Reply:
Fixed.